src/coreclr/pal/src/CMakeLists.txt

```diff
@@ -47,6 +47,8 @@ elseif(CLR_CMAKE_HOST_ARCH_ARM)
   set(PAL_ARCH_SOURCES_DIR arm)
 elseif(CLR_CMAKE_HOST_ARCH_ARM64)
   set(PAL_ARCH_SOURCES_DIR arm64)
+elseif(CLR_CMAKE_HOST_ARCH_LOONGARCH64)
```

Comment:
Unless this results in build breakage, I would move changes from this file to https://github.com/dotnet/runtime/pull/62887 that way each PR would be self-contained.

Reply:
Thanks.
Had moved.